### PR TITLE
fixed issue-5126

### PR DESCRIPTION
**Pull request by abhinaisai2002**
Fixes the issue #5126 
Added the intended labels to the tag from the issue.
Lint :+1: 
Test :+1:

### CODE REVIEW

**Comment by abhinaisai2002** [comment]
@sneridagh 
Can you help me in this?

**Review by sneridagh — CHANGES_REQUESTED** [review]
@abhinaisai2002 If you signed alteady the contributor agreement we are good, and it can be merged.

You pushed a npm lock file, please remove it. The changelog need to be only one line, take a look at other PRs as reference. Also please read the contributor guide in plone.org.

@Wagner3UB could you please review the changes?

**Comment by abhinaisai2002** [comment]
> @abhinaisai2002 If you signed alteady the contributor agreement we are good, and it can be merged. If not, you need to follow the instructions in the contributor guide.
> 
> You pushed a npm lock file, please remove it. The changelog need to be only one line, take a look at other PRs as reference. Also please read the contributor guide in plone.org.
> 
> @Wagner3UB could you please review the changes?

I have already sign the document actually, but the error was still showing in the workflow checks

**Review by abhinaisai2002 — COMMENTED** [review]
Did the review changes, all tests were passed

**Comment by abhinaisai2002** [comment]
@stevepiercy The error is still showing even if i am a member of this organization. How to get rid of this issue,

**Comment by stevepiercy** [comment]
@abhinaisai2002 it sounds like the emails on your Plone Contributor Agreement and GitHub do not match. Please follow @mister-roboto's helpful advice to resolve it.

File: src/components/manage/Sharing/Sharing.jsx [hunk 1]
```diff
@@ -412,10 +412,17 @@ class SharingComponent extends Component {
                           )}
                           {typeof entry.roles[role.id] === 'boolean' && (
                             <Checkbox
+                              id={`make-logged-in-users-as${role.id}${
+                                entry.roles[role.id] ? '-checked' : ''
+                              }`}
+                              name={`make-logged-in-users-as${role.id}${
+                                entry.roles[role.id] ? '-checked' : ''
+                              }`}
```
Comment (Wagner3UB) [inline src/components/manage/Sharing/Sharing.jsx:420]: Don't use "logged-in-users" as a fixed id/name; instead, use "role.id" to identify them correctly

File: src/components/manage/Sharing/Sharing.jsx [hunk 2]
```diff
@@ -412,10 +412,17 @@ class SharingComponent extends Component {
...
                               onChange={this.onChange}
                               value={`${entry.id}:${role.id}`}
                               checked={entry.roles[role.id]}
                               disabled={entry.login === this.props.login}
+                              aria-labelledby={`make-logged-in-users-as${role.id}`}
```
[the remark this answers is not in the document]
Comment (abhinaisai2002) [reply src/components/manage/Sharing/Sharing.jsx:425]: yeah but there is no proper space at the check boxes to show the label

**Pull request closed without merging**